source/main.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/env python
```

Comment:
good that you've added a main.py. You can take a look at this one for reference for the comments below: https://github.com/ninalopatina/Rocket_RL/blob/master/python/main.py